lib/business/calendar.rb

```diff
@@ -19,13 +19,14 @@ def self.load(calendar)
       raise "No such calendar '#{calendar}'" unless directory
 
       yaml = YAML.load_file(File.join(directory, "#{calendar}.yml"))
-      valid_keys = ['holidays', 'working_days']
+      valid_keys = %w(holidays working_days working_dates)
```

Comment:
I'd prefer to call this new parameter something like `extra_working_dates`, to make it more clearly different from the existing `working_days` parameter.